GitUI/Infrastructure/Telemetry/AppStartDiagnosticsReporter.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using GitCommands;
+
+namespace GitUI.Infrastructure.Telemetry
+{
+    public sealed class AppStartDiagnosticsReporter
+    {
+        private readonly ISshPathLocator _sshPathLocator = new SshPathLocator();
+
+        public void Report()
+        {
+            string sshClient;
+            var sshPath = _sshPathLocator.Find(AppSettings.GitBinDir);
+            if (string.IsNullOrEmpty(sshPath))
+            {
+                sshClient = "OpenSSH";
+            }
+            else if (GitCommandHelpers.Plink())
+            {
+                sshClient = "PuTTY";
```

Comment:
This should be logging the path to find what PuTTY (bundled/external) or what external client that is used.

Reply:
> Are you able to help? 

Will look at this tonight. Maybe just internal/external putty is sufficient.